ci-operator/templates/cluster-launch-e2e-openshift-ansible.yaml

```diff
@@ -69,103 +69,6 @@ objects:
 
     containers:
 
-    # Once admin.kubeconfig exists, executes shared tests
```

Comment:
This doesn't seem like a vars update

Reply:
Moved this to https://github.com/openshift/release/pull/1496